en/flight_controller/cubepilot_cube_yellow.md

```diff
@@ -0,0 +1,141 @@
+# Cube Yellow Flight Controller
+
+The [Cube Yellow](http://www.proficnc.com/61-system-kits2) flight controller is a flexible autopilot intended primarily for manufacturers of commercial systems. 
+
+![Cube Yellow](../../assets/flight_controller/cube/yellow/cube_yellow_hero.jpg)
+
+> **Tip** The manufacturer [Cube Docs](https://docs.cubepilot.org/user-guides/autopilot/the-cube-module-overview) contain detailed information, including an overview of the [Differences between Cube Colours](https://docs.cubepilot.org/user-guides/autopilot/the-cube-module-overview#differences-between-cube-colours).
+
+The controller is designed to be used with a domain-specific carrier board in order to reduce the wiring, improve reliability, and ease of assembly.
+For example, a carrier board for a commercial inspection vehicle might include connections for a companion computer, while a carrier board for a racer could includes ESCs for the frame of the vehicle.
+
+Cube includes vibration isolation on two of the IMU's, with a third fixed IMU as a reference / backup.
+
+> **Note** This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+
+## Key Features
+
+* 32bit STM32F777 (32bit [ARM Cortex M7](https://en.wikipedia.org/wiki/ARM_Cortex-M#Cortex-M7), 400 MHz, Flash 2MB, RAM 512 KB).
```

Comment:
From the board config it's an STM32F765II

Reply:
Pretty sure that's a bug.

Reply:
https://github.com/PX4/Firmware/pull/15414